lib/patron.rb

```diff
@@ -12,6 +12,7 @@ def self.update!
     end
 
     def self.sync_groups
+      @local_users[RailsMultisite::ConnectionManagement.current_db] = []
```

Comment:
I think we should just remove the cache completely. We're purging before the method that sets the cache which is really weird.